pom.xml

```diff
@@ -135,6 +141,19 @@
         <module>hawkular-alerts-actions</module>
       </modules>
     </profile>
+
+    <profile>
+      <id>openshift</id>
+      <properties>
+        <skipTests>true</skipTests>
```

Comment:
In general, I am against adding a skipTests inside a profile.
If this profile will be used to package things specific for openshift, testing is necessary.
Skipping tests can be added at command line.

What is the context of this profile ? It is some developer helper or it is part of other automated process ?